node/sc/sub_bridge_handler.go

```diff
@@ -138,6 +158,25 @@ func (sbh *SubBridgeHandler) getChildOperatorBalance() (*big.Int, error) {
 	return sbh.subbridge.localBackend.BalanceAt(ctx, sbh.subbridge.bridgeAccounts.cAccount.address, nil)
 }
 
+// getChildBridgeContractBalance returns the child bridge contract's balance.
+func (sbh *SubBridgeHandler) getChildBridgeContractBalance(addr common.Address) (*big.Int, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), timeout)
+	defer cancel()
+
+	found := false
+	for _, bridgeContractAddr := range sbh.subbridge.bridgeManager.GetAllChildBridgeAddrs() {
```

Comment:
Do we really make a slice for this function? What about adding address check logic into `bridgeManager`?
Then, we can call like `bridgeManager.IsInChildBridge(addr)`?

Reply:
Refactored. PTAL.